benchmarks/synth-bm/justfile

```diff
@@ -1,4 +1,5 @@
-neard := "../../target/release/neard"
+# Place the `neard` binary to be benchmarked here.
+neard := "./neard"
```

Comment:
nit:

I think you could also use something like
```
neard := env('NEARD_PATH', "./neard")
```
to read the path from env variables, if present